.sync.yml

```diff
@@ -9,6 +9,19 @@ appveyor.yml:
       CHECK: metadata_lint
     - RUBY_VERSION: 24-x64
       CHECK: rubocop
+    - RUBY_VERSION: 24
```

Comment:
I think you don't need most of these defined for appveyor in sync.yml. The template adds Puppet 5 with 24 and 24x64 testing and Puppet 4 with 21 and 21x64 testing by default and your additional ones here are replicating builds I think, is why there's so many runs in your appveyor :)

Reply:
Nice catch! Updated and now running a matrix of 9 rather than 11 in appveyor.